test/OpenTelemetry.Instrumentation.W3cTraceContext.Tests/W3CTraceContextTests.cs

```diff
@@ -88,6 +88,10 @@ public void W3CTraceContextTestSuiteAsync(string value)
         {
             Assert.StartsWith("FAILED (failures=3)", lastLine);
         }
+        else if (AspNetCoreHostingVersion.Major == 8)
+        {
+            Assert.StartsWith("FAILED (failures=40)", lastLine);
```

Comment:
@xiang17 - Could you further investigate to check what is the difference between `net8.0` and other versions. Thats lot of failures.

Reply:
It might not be trivial to fix. Can we complete this PR first? The issue can be kept open to track the issue.

Reply:
@xiang17, I do not think that Vishwesh is asking for the fix in the scope of the PR, but it will be great to find what is the root cause.

Reply:
@xiang17 - Yes, just looking for the root cause.

Reply:
I've figured it out. Apparently .NET 8 on Linux changed the default port for ASP.NET Core (but not changed on Windows, which explained why local Windows works but not in Docker).